Skills/Perk/FertilityDesireToBreed.gd

```diff
@@ -0,0 +1,27 @@
+extends PerkBase
+
+func _init():
+	id = Perk.FertilityDesireToBreed
+	skillGroup = Skill.Fertility
+
+func getVisibleName():
+	return "Desire to breed"
+
+func getVisibleDescription():
+	return "Your desire to be bred is so high that new life finds its way inside you with ease. \n" \
+			+ "While not pregnant you deal +30% lust damage and become more susceptible to the lust yourself. \n" \
+			+ "Significantly increases the chances of condoms breaking inside you." #450% to break results in 90% for normal and 4.5% for quality condoms 
```

Comment:
I would remove the lust buffs/debuffs and just keep the condoms breaking. Could reword the description so its about the player purposfully sabotaging condoms because right now its kinda magical and doesn't explain anything